### PR TITLE
Add description of authors

### PR DESCRIPTION
**Pull request by awfrancisco**
Adds description of guest author

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/awfrancisco-patch-1)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/18f.gsa.gov/awfrancisco-patch-1/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/awfrancisco-patch-1E/README.md)

### CODE REVIEW

File: _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ tags:
 excerpt: "The Technology Transformation Service (TTS) has been working with the Forest Service since August 2016 on what will eventually be an agency-wide system to process permit applications. TTS is primarily helping the Forest Service hire private vendors for this work, rather than build it ourselves, and one way we’re improving that process is through “discovery research.” "
 ---
 
+*Rebecca and Colin work here at TTS on our Acquisitions team. Aaron Burk is a Senior Project Manager at the Forest Service and the Product Owner for our engagement.
```
Comment (gemfarmer) [inline _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md]: @awfrancisco Is this supposed to be bolded or have an asterisk?

Comment (gemfarmer) [inline _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md]: And do we want to link to Rebecca and Colin's author pages?

Comment (awfrancisco) [inline _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md]: Italics. Must have lost something in the copy paste. And their author pages are linked directly above, so I don't think we need to link again here.

Comment (gemfarmer) [inline _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md]: Gotcha. Once it rebuilds, I'll merge if they italicize. I can't remember if one * is **bold** or _italics_

**Pull request merged**